<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-06T14:31:44Z" name="Default suite" finished-at="2024-06-06T14:32:03Z" duration-ms="18663">
    <groups>
      <group name="cucumber">
        <method signature="io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group>
    </groups>
    <test started-at="2024-06-06T14:31:44Z" name="Default test" finished-at="2024-06-06T14:32:03Z" duration-ms="18663">
      <class name="cucumber.TestNGRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)" started-at="2024-06-06T14:31:44Z" name="setUpClass" finished-at="2024-06-06T14:31:45Z" duration-ms="801" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@39ee94de]]>
              </value>
            </param>
          </params>
        </test-method>
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)" started-at="2024-06-06T14:31:45Z" name="runScenario" description="Runs Cucumber Scenarios" finished-at="2024-06-06T14:31:51Z" duration-ms="6299" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Negative Testing with incorrect credantials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Purchase the order from E-Commerce website"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Incorrect email or password.> but was:<Incorrect emaill or password.>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Incorrect email or password.> but was:<Incorrect emaill or password.>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:171)
	at org.testng.Assert.assertEquals(Assert.java:181)
	at Brindha.Maven_Project.StepDefinition.stepdefinition.error_message_displayed_on_confirmation_page(stepdefinition.java:67)
	at ✽."Incorrect emaill or password." message on login page(file:///C:/Users/Brindha/eclipse-workspace/newworkspace/Maven_Project/src/test/java/cucumber/ErrorValidation.feature:9)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)" started-at="2024-06-06T14:31:51Z" name="runScenario" description="Runs Cucumber Scenarios" finished-at="2024-06-06T14:32:03Z" duration-ms="11490" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Positive test of submitting the order"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Purchase the order from E-Commerce website"]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at Brindha.Maven_Project.StepDefinition.stepdefinition.message_displayed_on_confirmation_page(stepdefinition.java:61)
	at ✽."THANKYOU FOR THE ORDER" the message is displayed on confirmation page(file:///C:/Users/Brindha/eclipse-workspace/newworkspace/Maven_Project/src/test/java/cucumber/purchaseorder.feature:13)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="tearDownClass()" started-at="2024-06-06T14:32:03Z" name="tearDownClass" finished-at="2024-06-06T14:32:03Z" duration-ms="54" status="PASS">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
